src/main/java/com/puppycrawl/tools/checkstyle/checks/SuppressWarningsHolder.java

```diff
@@ -86,19 +103,48 @@
  * the {@code aliasList}:
  * </p>
  * <pre>
- * &#64;SuppressWarnings("paramnum")
- * public void needsLotsOfParameters(@SuppressWarnings("unused") int a,
- *   int b, int c, int d, int e, int f, int g, int h) {
- *   ...
+ * &lt;module name=&quot;TreeWalker&quot;&gt;
+ *   &lt;module name=&quot;ParameterNumber&quot;/&gt;
+ *   &lt;module name=&quot;SuppressWarningsHolder&quot;&gt;
+ *     &lt;property name=&quot;aliasList&quot; value=
+ *       &quot;com.puppycrawl.tools.checkstyle.checks.sizes.ParameterNumberCheck=paramnum&quot;/&gt;
+ *   &lt;/module&gt;
+ * &lt;/module&gt;
+ * &lt;module name=&quot;SuppressWarningsFilter&quot;/&gt;
+ * </pre>
+ * <pre>
+ * class Test {
+ *
+ *     public void needsLotsOfParameters(@SuppressWarnings("unused") int a, // violation
+ *       int b, int c, int d, int e, int f, int g, int h) {
+ *       ...
+ *     }
+ *
+ *     &#64;SuppressWarnings("paramnum")
+ *     public void needsLotsOfParameters1(@SuppressWarnings("unused") int a, // violation suppressed
```

Comment:
please remove `@SuppressWarnings("unused")` to let user clearly see what is suppressed.

Reply:
done